README.md

```diff
@@ -181,29 +181,52 @@ it into the "Index pattern" field, then click the "Next step" button.
 Discover section.
 
 
-#### Using Docker
+#### Using Docker Locally
 
-1. Download, install, and launch [Docker]
+1. Download, install, and launch [Docker](https://www.docker.com/products/docker-desktop)
 
-1. Set up the Docker image
+1. Build and launch the Docker containers: `docker-compose up`
 
-  ```
-  $ bin/setup --docker
-  ```
+1. Copy necessary config files locally. This dir is mounted as a volume, so new files also appear in the container.
+
+```sh
+cp config/application.yml.default config/application.yml
+cp -R certs.example certs
+cp -R keys.example keys
+cp pwned_passwords/pwned_passwords.txt.sample pwned_passwords/pwned_passwords.txt
+cp config/service_providers.localdev.yml config/service_providers.yml
+```
+
+1. Manually edit `application.yml`, in the `development:` section to make Redis look like
 
-[Docker]: https://docs.docker.com/engine/getstarted/step_one/#step-1-get-docker
+```sh
+  redis_throttle_url: redis://redis:6379
+  redis_url: redis://redis:637
+```
+
+1. Bootstrap the databases
+
+```sh
+docker-compose run --rm web rake db:create
```

Comment:
what if we put these in a `bin/docker-bootstrap` or `bin/docker-setup` script?

Reply:
yep.  pushing that in a bit.